docs/sources/tempo/operations/manage-advanced-systems/user-configurable-overrides.md

```diff
@@ -98,12 +102,13 @@ If the tenant is run in distributed mode, only the query-frontend will accept AP
 Returns the current overrides and it's version.
 
 Query-parameters:
+
 - `scope`: whether to return overrides from the API only `api` or merge it with the runtime overrides `merged`. Defaults to `api`.
 
 Example:
 
 ```shell
-$ curl -X GET -v -H "X-Scope-OrgID: 3" http://localhost:3100/tempo/api/overrides\?scope=merged`
+curl -X GET -v -H "X-Scope-OrgID: 3" http://localhost:3100/tempo/api/overrides\?scope=merged`
```

Comment:
Is the ` at the end of the example supposed to be there?

Reply:
good catch! I don't think this should be there.